src/main/java/io/cryostat/net/SslConfiguration.java

```diff
@@ -64,9 +64,12 @@ public class SslConfiguration {
         this.logger = logger;
 
         if (env.hasEnv(Variables.DISABLE_SSL)) {
```

Comment:
How about just changing `env.hasEnv(Variables.DISABLE_SSL))` to `env.getEnv(Variables.DISABLE_SSL, "").equals("true")`?

Reply:
This solution is better by getting the value of the env variable and then see if its value is equal to "true"

Reply:
I think that's pretty similar to what my suggestion is doing, but if you'd prefer to use `hasEnv` and then `getEnv`, could you combine them into a single `if` condition?

Reply:
this is what I have 
 ```String disableSslValue = env.getEnv(Variables.DISABLE_SSL,"");
            if (disableSslValue.equals("true")) {
                strategy = new NoSslStrategy();
                logger.info("Selected NoSSL strategy");
                return;
            }
         ``` 
so far, this works with the idea that you had. This looks good in my opinion than using both ```hasEnv``` and ```getEnv```

Reply:
yes... and I think this makes more sense

Reply:
my idea of ```hasEnv``` is kinda long cut where as your idea of ```getEnv``` will do both the check and retrieving the value

Reply:
```if ("true".equals(env.getEnv(Variables.DISABLE_SSL))) {
    strategy = new NoSslStrategy();
    logger.info("Selected NoSSL strategy");
    return;
}```
This looks cleaner?